.pre-commit-config.yaml

```diff
@@ -7,6 +7,7 @@ repos:
         name: ruff
         description: "Run 'ruff' for linting"
         args: ["--fix"]
+        exclude: docs/ 
```

Comment:
? what was this for?

Reply:
I think I had some linting issues with docs before, but since it seems to pass in PR checks I guess it was only on my end. removing 👍🏼 
```suggestion
```